include/circt/Dialect/Calyx/CalyxPrimitives.td

```diff
@@ -267,6 +267,10 @@ def AndLibOp  : CombinationalArithBinaryLibraryOp<"and"> {}
 def OrLibOp   : CombinationalArithBinaryLibraryOp<"or"> {}
 def XorLibOp  : CombinationalArithBinaryLibraryOp<"xor"> {}
 
+def MuxLibOp : CalyxLibraryOp<"mux", [Combinational, SameTypeConstraint<"tru", "fal">, SameTypeConstraint<"tru", "out">]> {
```

Comment:
nit: line length, i would usually split the trait list to the next line.

Reply:
Done!